metrics/README.md

```diff
@@ -131,9 +131,10 @@ we provide a bash API to aid storing results in a uniform manner.
 
 The preferred API to store results is through the provided JSON API.
 
-The API provides two groups of functions:
+The API provides the following groups of functions:
 - A set of functions to init/save the data and add 'top level' JSON fragments
 - A set of functions to construct arrays of JSON fragments, which are then added as a top level fragment when complete
+- A set of functions to construct elements of an array from sub-fragments, and then finalize that element when all fragments have been added.
```

Comment:
Suggested rewrite to stay active: 
A set of functions to construct elements of an array from sub-fragments, and then finalize that element when all fragments are added.